src/components/portfolioComponents/portfolioOverallViz/PortfolioOverallViz.js

```diff
@@ -0,0 +1,123 @@
+import { Container, Card, Row, Col, Dropdown } from "react-bootstrap";
+import { useState } from "react";
+import PortfolioGraph from "../portfolioGraph/portfoliograph";
+
+function PortfolioOverallViz() {
+
+    const [portfolioName, setPortfolioName] = useState("PortfolioA")
+
+    // dummy data of portfolio history
+    const PortfolioA = [
+        { date: '01-10', value: 400 },
+        { date: '01-11', value: 700 },
+        { date: '01-12', value: 60 },
+        { date: '01-13', value: 700 },
+        { date: '01-14', value: 500 },
+        { date: '01-15', value: 400 },
+        { date: '01-16', value: 700 },
+        { date: '01-17', value: 60 },
+        { date: '01-18', value: 1000 },
+        { date: '01-19', value: 500 },
+        { date: '01-20', value: 400 },
+        { date: '01-21', value: 700 },
+        { date: '01-22', value: 60 },
+        { date: '01-23', value: 700 },
+        { date: '01-24', value: 500 }
+    ]
+    const PortfolioB = [
+        { date: '01-10', value: 400 },
+        { date: '01-11', value: 700 },
+        { date: '01-12', value: 60 },
+        { date: '01-13', value: 200 },
+        { date: '01-14', value: 500 },
+        { date: '01-15', value: 10 },
+        { date: '01-16', value: 700 },
+        { date: '01-17', value: 60 },
+        { date: '01-18', value: 700 },
+        { date: '01-19', value: 500 },
+        { date: '01-20', value: 400 },
+        { date: '01-21', value: 700 },
+        { date: '01-22', value: 60 },
+        { date: '01-23', value: 700 },
+        { date: '01-24', value: 300 },
+    ]
+    const PortfolioC = [
+        { date: '01-10', value: 100 },
+        { date: '01-11', value: 200 },
+        { date: '01-12', value: 60 },
+        { date: '01-13', value: 300 },
+        { date: '01-14', value: 500 },
+        { date: '01-15', value: 10 },
+        { date: '01-16', value: 700 },
+        { date: '01-17', value: 60 },
+        { date: '01-18', value: 700 },
+        { date: '01-19', value: 800 },
+        { date: '01-20', value: 100 },
+        { date: '01-21', value: 200 },
+        { date: '01-22', value: 630 },
+        { date: '01-23', value: 900 },
+        { date: '01-24', value: 100 },
+    ]
+    // simulates a list of portfolio names attached to user. Basic logic in but will have to change with real data.
+    const portfolioList = ["PortfolioA", "PortfolioB", "PortfolioC"]
+
+    const [data, setData] = useState(PortfolioA);
+
+    const handleSelect = (e) => {
+        if (e.target.id === "PortfolioA") {
```

Comment:
For these I know its dummy data but just in general if you find yourself writing 2+ of same if statement try to break it into a loop, I think the portfolios will be in array more so too. 

Pseudo code of what I mean below. 
```
portfolios = [
    {"name":"portfolioA", "values":[{"date":"dateVal","value":"$"}, ....]},
    {"name":"portfolioB", "values":[{"date":"dateVal","value":"$"}, ....]},
    {"name":"portfolioC", "values":[{"date":"dateVal","value":"$"}, ....]}
]

for portfolio in portfolios{
   if e.target.id === portfolio.name{
     setYourState
    }
}
```